dashboards/use.libsonnet

```diff
@@ -74,9 +74,15 @@ local g = import 'grafana-builder/grafana.libsonnet';
         g.row('Storage')
         .addPanel(
           g.panel('Disk Capacity') +
-          g.queryPanel(|||
-            sum(max(node_filesystem_size{fstype=~"ext[24]"} - node_filesystem_avail{fstype=~"ext[24]"}) by (device,%(podLabel)s,namespace)) by (%(podLabel)s,namespace) / scalar(sum(max(node_filesystem_size{fstype=~"ext[24]"}) by (device,%(podLabel)s,namespace))) * on (namespace, %(podLabel)s) group_left(node) node_namespace_pod:kube_pod_info:
-          ||| % $._config, '{{node}}', legendLink) +
+          g.queryPanel(
+            |||
+              sum(max(node_filesystem_size{%(fstypeSelector)s} - node_filesystem_avail{%(fstypeSelector)s}) by (device,%(podLabel)s,namespace)) by (%(podLabel)s,namespace)
```

Comment:
I don't think the `max` is needed here

Reply:
I actually didn't change this expressions, just added a few newlines for better readability so I didn't question the correctness, just added `fstypeSelector` references.

But: As far as I understand the `max` grouping removes duplicates caused by bind mounts, it could be a `min` too.

Reply:
My brain is melting on this one as well, I think the `max(...) by (...)` should be just around the size metric the subtraction does not happen on non existent vector entries of the left side, but tbh I would have to check myself.

Reply:
as discussed out of band: let's revisit this one in a subsequent PR. Currently this mechanically just adds the filesystem type selector.

Generally: The disk capacity calculation here is currently per node. The problem is that, on each node, we additionally have multiple devices, so a "per node" disk space calculation in the cluster view is indeed quite questionable anyways.